ts/components/InstabugAssistanceComponent.tsx

```diff
@@ -40,7 +44,11 @@ const InstabugAssistanceComponent: React.FunctionComponent<Props> = ({
       <View spacer={true} />
       <ButtonWithImage
         icon={"io-bug"}
-        onPress={() => requestAssistance(BugReporting.reportType.bug)}
+        onPress={
```

Comment:
`InstabugAssistanceComponent.tsx` should not known nothing about the token.
Its responsibility should be "someone press me and ask for assistance about bug"

A better design could be:

- `ContextualHelpModal` uses `ContenxualHelpHome` (or `ContenxualHelpComponent`) and `SendSupportTokenInfo` (this already the case)
- `ContenxualHelpHome` exposes an handler `onRequestAssistance`
- `SendSupportTokenInfo` exposes an handler `onContinue` to know if the user accept or not
- `ContextualHelpModal` uses these handler and keeps the state

check my attempt a3cf60ab2e53791366ac3cab4b03e3d242aa3118